packages/sitecore-jss-nextjs/src/middleware/multisite-middleware.ts

```diff
@@ -89,21 +53,18 @@ export class MultisiteMiddleware {
     // Response will be provided if other middleware is run before us
     let response = res || NextResponse.next();
 
-    if (
-      this.excludeRoute(pathname) ||
-      (this.config.excludeRoute && this.config.excludeRoute(pathname))
-    ) {
+    if (this.excludeRoute(pathname)) {
```

Comment:
We should check `isPreview` and skip for multisite middleware as well. Think we've been OK since the editors pass a "sc_site" query string (avoiding use of `siteResolver.getByHost`) but there's really no need to run this in editing context and would cause issues if editors ever remove that.